Voycar.Api.Web/Features/Members/Endpoints/Get/Personal/Endpoint.cs

```diff
@@ -0,0 +1,44 @@
+namespace Voycar.Api.Web.Features.Members.Endpoints.Get.Personal;
+
+using Repository;
+using Users.Repository;
+
+
+public class Endpoint : Endpoint<Request, Results<Ok<Response>, BadRequest<NotFound>>, Mapper>
+{
+    private readonly IUsers _userRepository;
+    private readonly IMembers _memberRepository;
+
+    public Endpoint(IUsers userRepository, IMembers memberRepository)
+    {
+        this._userRepository = userRepository;
+        this._memberRepository = memberRepository;
+    }
+
+
+    public override void Configure()
+    {
+        this.Get("member/personal");
+        this.Roles("member");
+        this.Summary(summary =>
+        {
+            summary.Summary = "Get personal details of logged in member";
+            summary.Description = "Get the personal details of the member who is logged in according to the request cookie";
+            summary.Responses[200] = "If member is found";
+            summary.Responses[400] = "If the member somehow can't be found";
+        });
+    }
+
+    public override async Task HandleAsync(Request req, CancellationToken ct)
+    {
+        var user = this._userRepository.Retrieve(req.UserId);
+        if (user is null)
+        {
+            await this.SendResultAsync(TypedResults.BadRequest("User does not exist"));
```

Comment:
Wenn im Swagger steht, die Fehlermeldung kommt, wenn es den member nicht gibt, sollten wir hier auch die Fehlermeldung entsprechend ausgeben. (oder halt Swagger anpassen)
```suggestion
            await this.SendResultAsync(TypedResults.BadRequest("Member does not exist"));
```

Reply:
hm, würde ich so lassen, weil ich da prüfe, ob es den User gibt

Reply:
summary.Responses[400] = "If the member somehow can't be found or the user does not exists / is not a member."; dann eher sowas

Reply:
1. Ja hast recht, Fehlermeldung lassen wir so 
2. Für Doku hätte ich eher gesagt: "If the user somehow can't be found or the user is not a member." Weil sonst sieht es so aus, als gibt es drei mögliche Gründe für den Statuscode, obwohl es nur zwei gibt.
3. Das hier wird noch gemacht, oder nicht? https://github.com/solid-stack-solutions/voycar-backend/pull/56#discussion_r1648831352

Reply:
Wie gesagt, ich weiß nicht was jannik alles braucht

Reply:
Hab da noch nen null check reingepackt